src/components/dashboard/documents-tab.yml

```diff
@@ -0,0 +1,15 @@
+documentsTab:
+  info:
+    en: Here you can find the public documents you have created or contributed to.
+    de: Hier findest du die öffentliche Dokumente, die du erstellt oder bearbeitet hast.
+  noDocuments:
+    en: You have not contributed to any public documents yet.
+    de: Du hast noch keine öffentliche Dokumente bearbeitet.
+  ownDocumentOnly:
+    en: My documents only
+    de: Nur meine Dokumente
```

Comment:
Do you think it is clear that "my" means, "created by me"? We could be more verbous here and have "Only documents that I have created" / "Nur Dokumente, die ich erstellt habe"

Reply:
True, changed.